doc/spec/namespace.md

```diff
@@ -0,0 +1,148 @@
+# Namespace
+The global namespace represents the full set of named repositories which can be
```

Comment:
Forgive me for my dumb questions which may be out of scope of this document. I'd just like to get a picture. Is this the *global federated namespace* - the one and only public namespace out there? Is it supposed to be an instance of distribution? Whose responsibility is a creation of namespaces and a publishing of repository inside it (registry/client/user through some web UI)?